lib/doorkeeper/config.rb

```diff
@@ -510,6 +530,9 @@ def calculate_authorization_response_types
       types = []
       types << "code"  if grant_flows.include? "authorization_code"
       types << "token" if grant_flows.include? "implicit"
+      if grant_flows.include? "urn:ietf:params:oauth:grant-type:device_code"
```

Comment:
I believe we have some module with such constants, like OOB NATIVE REDIRECT URI or smth like this. Could you please check and move it there?